GOVERNANCE.md

```diff
@@ -75,46 +75,37 @@ Charter need approval by the Node.js Board of Directors.
 
 ### TSC Meetings
 
-The TSC meets regularly in a voice conference call. The meeting is run by a
-designated meeting chair approved by the TSC. Each meeting is streamed on
-YouTube.
+The TSC meets in a voice conference call. Each year, the TSC elects a chair to
+run the meetings. The TSC streams its meetings for public viewing on YouTube or
+a similar service.
 
-Items are added to the TSC agenda which are considered contentious or
-are modifications of governance, contribution policy, TSC membership,
-or release process.
+The TSC agenda consists only of issues that are at an impasse. The intention of
```

Comment:
This change now excludes things like https://github.com/nodejs/node/issues/26186 which was an example of `modifications of TSC membership`. Given that voting for approving TSC members happens in the TSC meetings it kind of makes sense for those issues to be added to the agenda.

Reply:
> Given that voting for approving TSC members happens in the TSC meetings it kind of makes sense for those issues to be added to the agenda.

I'm OK with adding it back if that's the consensus, but IIUC, the *only* reason that stuff was in the agenda was because this doc (and maybe others?) said or implied that it had to happen in a meeting. I suspect current TSC consensus would be that there's no reason for the vote to have to happen in a meeting. It can (and, IMO, should) happen in a GitHub issue. The new member can be announced/celebrated during the announcements portion of the meeting.

Voting during meetings is suboptimal and should be avoided IMO because it means only people who can attend that particular meeting get to vote. For membership votes (and some others), we subverted that by gathering votes ahead of time. But the gathering of votes ahead of time, especially for a non-controversial nomination, would seem to argue that the vote doesn't need to happen in the meeting.

Reply:
I'll defer to @nodejs/tsc if this covers everything that should be on the agenda. Looking at the recent agenda I see tracking issues and things related to the merger of the foundations which all appear to be for informational/discussion purposes and are not things at an impasse.

Reply:
I don't think its quite right to say the agenda `only` consists of things that are at impasse.  An update for the board and the strategic initiatives don't fall under that.

Reply:
The tracking issues are things I have argued should be part of the meeting template (like Announcements) and not individual agenda items. If this lands, I'll open a PR against the meeting agenda generator to make that so.

IMO, the informational things are OK if they are not getting sufficient attention. Perhaps rather than "impasse", it say something about being stalled due to disagreement or lack of attention? Or maybe "impasse" covers it sufficiently, if somewhat imprecisely?

Reply:
@richardlau @mhdawson I changed the text to be less rigid: "The TSC agenda includes issues that are at an impasse." That doesn't exclude anything. Let me know if that doesn't sufficiently address your concerns. Thanks!